submissions/array_sort_c/README.md

```diff
@@ -0,0 +1,12 @@
+Project Name: array_sort_c
+
+Contributors: Devin Forrest
+
+How to Run: Compile with gcc then run the output file
+            gcc sort.c -o sort.o
+
+What it Does: The program first asks the user to provide the size of the array that they want to sort
+              The program then prompts the user for the values in the array to sort once at a time
+              The program then uses qsort to sort the array then prints the sorted array
+
+Why I contributed it: It is a good example of the usage of qsort
```

Comment:
Try to use markdown formatting for the readme, it makes it quite nice to read and show up nicely in the UI when it gets rendered. [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) is some more information about markdown